package.json

```diff
@@ -27,21 +27,23 @@
   "dependencies": {
     "@react-navigation/drawer": "^6.1.8",
     "@react-navigation/native": "^6.0.6",
-    "expo": "~43.0.2",
-    "expo-linking": "^2.4.2",
-    "expo-status-bar": "~1.1.0",
+    "expo": "^46.0.9",
+    "expo-linking": "~3.2.2",
+    "expo-modules-core": "^0.11.4",
+    "expo-status-bar": "~1.4.0",
     "moti": "^0.16.1",
     "native-base": "^3.2.1",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
-    "react-native": "0.64.3",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-native": "^0.69.5",
     "react-native-checkbox-reanimated": "^0.1.0",
-    "react-native-gesture-handler": "^1.10.3",
-    "react-native-reanimated": "^2.2.4",
-    "react-native-safe-area-context": "^3.3.2",
-    "react-native-screens": "^3.9.0",
-    "react-native-svg": "^12.1.1",
-    "react-native-web": "0.17.1",
+    "react-native-gesture-handler": "~2.5.0",
+    "react-native-reanimated": "~2.9.1",
+    "react-native-safe-area-context": "^4.3.1",
+    "react-native-screens": "~3.15.0",
+    "react-native-svg": "^12.3.0",
+    "react-native-web": "^0.18.8",
+    "react-stately": "^3.17.0",
```

Comment:
can you tell me why `react-stately` is added?

Reply:
Oh, I ran into an error while bundling the app. It said that one of the packages had react-stately as a dependency. The app wouldn't bundle without it. So I installed it to fix the error.